utils/fasm2fasm.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+'''
+Takes input FASM and outputs optionally canonical FASM.
```

Comment:
if you don't make it canonical, what is the tool doing? is this more testing that the API works? Seems if its to make it canonical, the tool should just do that, and do it well

Reply:
and API test should be a separate utility

Reply:
Think of this tool as a pretty printer.  I'm planning on adding modes for bit fusion, etc.  Maybe fasm_pretty.py or fasm_pretty_print.py or fasm_pprint.py is a better name?

Reply:
fasm_pretty or fasm_pprint sounds good to me